test/check-application

```diff
@@ -447,20 +462,69 @@ class TestNavigator(testlib.MachineCase):
         b.wait_not_present("[data-item='newdir1']")
 
         # Check file name is pre-selected when creating link
-        m.execute("touch /home/admin/newfile")
+        m.execute("echo 'some content' > /home/admin/newfile")
```

Comment:
Please ensure that the downloaded file has exactly this content.

Reply:
Done.